frontend/canvas-chrome-ext/src/scripts/inject_script.js

```diff
@@ -44,14 +44,14 @@
         });
     })
 
-
-
     btn_code_submission.addEventListener("click", function () {
```

Comment:
Can we standardize format of how we name variables? There's a mix of snake_case and camelCase. I think the standard for JS is camelCase if we wanna just do that to match the functions